accord-core/src/main/java/accord/coordinate/FetchMaxConflict.java

```diff
@@ -45,32 +46,34 @@ public class FetchMaxConflict extends AbstractCoordinatePreAccept<Timestamp, Get
 {
     final QuorumTracker tracker;
     final Seekables<?, ?> keysOrRanges;
+    final CommandStore store;
     Timestamp maxConflict;
     long executionEpoch;
 
-    private FetchMaxConflict(Node node, FullRoute<?> route, Seekables<?, ?> keysOrRanges, long executionEpoch)
+    private FetchMaxConflict(Node node, CommandStore store, FullRoute<?> route, Seekables<?, ?> keysOrRanges, long executionEpoch)
     {
-        this(node, route, keysOrRanges, executionEpoch, node.topology().withUnsyncedEpochs(route, executionEpoch, executionEpoch));
+        this(node, store, route, keysOrRanges, executionEpoch, node.topology().withUnsyncedEpochs(route, executionEpoch, executionEpoch));
     }
 
-    private FetchMaxConflict(Node node, FullRoute<?> route, Seekables<?, ?> keysOrRanges, long executionEpoch, Topologies topologies)
+    private FetchMaxConflict(Node node, CommandStore store, FullRoute<?> route, Seekables<?, ?> keysOrRanges, long executionEpoch, Topologies topologies)
     {
         super(node, route, null, topologies);
+        this.store = store;
         this.keysOrRanges = keysOrRanges;
         this.maxConflict = Timestamp.NONE;
         this.executionEpoch = executionEpoch;
         this.tracker = new QuorumTracker(topologies);
     }
 
-    public static AsyncResult<Timestamp> fetchMaxConflict(Node node, Seekables<?, ?> keysOrRanges)
+    public static AsyncResult<Timestamp> fetchMaxConflict(Node node, CommandStore store, Seekables<?, ?> keysOrRanges)
```

Comment:
why are you passing the `store`?  Shouldn't `CommandStore.current()` be the same thing?

Reply:
Most of the existing code passes around a `store` instead of using a thread local.

That said... the store is unused? I should be able revert this change entirely.